st2common/st2common/service_setup.py

```diff
@@ -165,9 +173,39 @@ def setup(service, config, setup_db=True, register_mq_exchanges=True,
 
     metrics_initialize()
 
+    # Register service in the service registry
+    if service_registry:
```

Comment:
Should we add a config check here?  If user specifies a backend with no service registry support, we log and warn users and start st2 w/o service registry?

Reply:
Did you address this?  Or is this a bad idea?

Reply:
We already log a warning when ``get_coordinator`` is called inside ``register_service_in_service_registry`` function.